NohanaImagePicker/AssetDateSectionHeaderView.swift

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2021 nohana, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an &quot;AS IS&quot; BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+
+protocol AssetDateSectionHeaderViewDelegate: AnyObject {
+    func didPushPickButton()
+}
+
+class AssetDateSectionHeaderView: UICollectionReusableView {
+    @IBOutlet weak private var dateLabel: UILabel!
+    @IBOutlet weak var pickButton: UIButton!
```

Comment:
```suggestion
    @IBOutlet weak private var pickButton: UIButton!
```
It would be better to set the access modifier to "private".